frontend/src/metabase/components/MetadataInfo/InfoIcon/index.tsx

```diff
@@ -0,0 +1 @@
+export * from "./InfoIcon.styled";
```

Comment:
The loading spinner is cut off

https://github.com/metabase/metabase/assets/6830683/2d79120b-5d1e-4569-80e8-6eafc630655d

Reply:
Good catch, seems like it was a bug already in `master`.

I've added a fixed here!